app/helpers/user_controller_helper.rb

```diff
@@ -6,4 +6,16 @@ def render_collaborators_for(user)
   def render_demo_logout_button_for(user)
     render "users/demo_logout_button" if user.is_demo_user?
   end
+
+  def resource_name
+    :user
+  end
+
+  def resource
+    @user
+  end
+
+  def devise_mapping
+    @devise_mapping ||= Devise.mappings[:user]
+  end
```

Comment:
It was a Devise thing that threw a method_missing error unless it was defined. This was the example in the Devise wiki:
https://github.com/plataformatec/devise/wiki/How-To:-Display-a-custom-sign_in-form-anywhere-in-your-app